web/pages/index.vue

```diff
@@ -24,42 +22,34 @@ watchDebounced(
   { debounce: 500 },
 );
 
-let components = new Map<string, Component>();
 
 onMounted(async () => {
   await activitiesStore.load(filter.value);
```

Comment:
I decided to move filter.value argument into the store and it is not used anymore, please see recent changes in main branch.